concepts/observability-strategy/configurable-logging/01-groundwork.md

```diff
@@ -35,15 +34,14 @@ This concept proposes how to open up to those new scenarios by making integratio
 ### Template instantiation
 - A template is instantiated by binding it to a secret that provides input for placeholders like URL and credentials. Alternatively, placeholders can be filled out with configMaps and static values.
   - The template validates whether all placeholders will be replaced with values.
-- A template can be instantiated for a specific workload and/or namespaces only.
-
-# Proposed Solution
+- A template can be instantiated for a specific workload and/or Namespaces only.
 
-The idea of the proposal is to introduce a preconfigured agent layer that's responsible for collecting all telemetry data. Those agents can be configured dynamically at runtime with different configuration scenarios, so that the agents start shipping the data to the configured backends. The dynamic configuration and management of the agent is handled by a new operator, which is configured using Kubernetes resources. The agents and the new operator are bundled in a new core package called `telemetry`. The existing Kyma backends and UIs will be just one possible solution that can be installed optionally and will stay in the existing modules like `logging`.
+### Local backend
+- Kyma will provide a blueprint based on Helm for installing the typical Loki stack as example.
+- The setup is not meant to be HA and scalable.
 
-![b](./assets/future_all.drawio.svg)
+# Proposed Solution
 
-# Focus on Logging
+The idea of the proposal is to introduce a preconfigured agent layer that's responsible for collecting all telemetry data. Users can configure those agents dynamically at runtime with different configuration scenarios, so that the agents start shipping the data to the configured backends. The dynamic configuration and management of the agent is handled by a new operator, which is configured with Kubernetes resources. The agents and the new operator are bundled in a new core package called `telemetry`. The existing Kyma backends and UIs will be just one possible solution to integrate with. The user can install them manually following a blueprint.
```

Comment:
```suggestion
The proposal introduces a new preconfigured agent layer that's responsible for collecting all telemetry data. Users can configure those agents dynamically at runtime with different configuration scenarios, so that the agents start shipping the data to the configured backends. The dynamic configuration and management of the agent is handled by a new operator, which is configured with Kubernetes resources. The agents and the new operator are bundled in a new core package called `telemetry`. The existing Kyma backends and UIs will be just one possible solution to integrate with. The user can install them manually following a blueprint.
```